README.md

```diff
@@ -8,6 +8,28 @@ Clementine is a modern music player and library organizer for Windows, Linux and
 - Buildbot: http://buildbot.clementine-player.org/grid
 - Latest developer builds: http://builds.clementine-player.org/
 
+Opening an issue
+----------------
+### Ask a new feature
+
+Please :
+
+ * Check if the new feature is not already implemented (Changelog)
+ * Check if another person didn't already open an issue
+ * If there is already an opened issue there is no need to comment "+1", it won't help :o)
```

Comment:
Maybe adding a "instead, you can subscribe to the issue to be notified of anything new about it" or something like this.